app/models/activity.rb

```diff
@@ -157,4 +157,12 @@ def transactions_total
   def planned_disbursements_total
     planned_disbursements.sum(:value)
   end
+
+  def actual_total_for_report(report:)
+    transactions.where(report: report, date: report.created_at.all_quarter).sum(:value)
+  end
+
+  def forecasted_total_for_report(report:)
+    planned_disbursements.where(report: report, period_start_date: report.created_at.all_quarter).sum(:value)
```

Comment:
To fully blow your mind (and possibly make you want to cry!):

Forecasted values for this quarter should not have been submitted in this report (they are from the past, we should not expect what was forecasted to change after we spent the money) so for this scope we only want the date range, as restricting to the report will in theory remove all the values we are interested in.

I would also like to name this one 'forecasted_total_for_report_financial_quarter' if you agree?